token-swap/program/src/processor.rs

```diff
@@ -597,7 +600,7 @@ mod tests {
 
     #[test]
     fn test_initialize() {
-        let swap_key = pubkey_rand();
+        let swap_key = Pubkey::from_str("7eLu8C6ZGATFZFXgzyem5oQaWMr2qTUBoHAbAoEQ5EZi").unwrap();
```

Comment:
This looks like a temporary bandaid fix, yes?

Reply:
I dunno, I've always perffered tests to run on static data

Reply:
What I mean is the usage model if this is an program address is to find one that's off the curve and pass the nonce.  That's not happening here so the test isn't mirroring the use case

Reply:
Sure.  I'll do it in a follow up tomorrow, though.  I've been trying to get a four line diff into the ledger app repo for ~20hrs now and am running out of steam